engines/registration_sharing/lib/registration_sharing/client.rb

```diff
@@ -21,7 +21,7 @@ def sync_system
   def peer_register_system(system)
     params = {}
 
-    %w[login password hostname proxy_byos system_token registered_at created_at last_seen_at instance_data].each do |attribute|
+    %w[login password hostname proxy_byos_mode system_token registered_at created_at last_seen_at instance_data].each do |attribute|
```

Comment:
Doesn't the client still need to receive `proxy_byos`?

Reply:
not necessarily, depending on how we handle the regsharing info

if we keep the column `proxy_byos` to be removed in a different iteration, then yes, we would need to send `proxy_byos` 

if we are [handling the info](https://github.com/SUSE/rmt/pull/1194#discussion_r1701587459) on the regsharing receiving side, then `proxy_byos_mode` should do